components/permissionsForm/index.test.jsx

```diff
@@ -21,15 +21,31 @@
 
 /* eslint-disable camelcase */
 import React from "react";
+import * as routerFns from "next/router";
 import PermissionsForm, {
   arrowIcon,
   permissionHandler,
   toggleOpen,
 } from "./index";
 import { ACL, createAccessMap } from "../../src/solidClientHelpers/permissions";
 import { renderWithTheme } from "../../__testUtils/withTheme";
+import useAuthenticatedProfile from "../../src/hooks/useAuthenticatedProfile";
+import { mockProfileAlice } from "../../__testUtils/mockPersonResource";
+
+jest.mock("../../src/hooks/useAuthenticatedProfile");
+const mockedAuthenticationProfile = useAuthenticatedProfile;
 
 describe("PermissionsForm", () => {
+  const authProfile = mockProfileAlice();
+  const visitorIri = "http://some-random-pod.com/";
+
+  beforeEach(() => {
+    jest
+      .spyOn(routerFns, "useRouter")
+      .mockReturnValue({ query: { iri: visitorIri } });
+    mockedAuthenticationProfile.mockReturnValue({ data: authProfile });
+  });
```

Comment:
Tests should be updated to reflect the updated `disabled` logic - such as a negative case.

Reply:
This is handled in https://github.com/inrupt/pod-browser/pull/243/files#diff-fb36631f81b590b8107939e2bc4400e75b8dff0f751d0d0a67f26973f157f40dR180-R194. I did consider whether to have a test directly in permissionsForm.test.jsx as well, but didn't think it was worth doing, since it's covered, and it's a component we'll phase out soon anyway.

But ok, I can write an extra test for it.

Reply:
Tests written